Sources/Neon/TextViewSystemInterface.swift

```diff
@@ -13,13 +13,13 @@ import Rearrange
 public typealias TokenAttributeProvider = (Token) -> [NSAttributedString.Key: Any]
 
 #if os(macOS) || os(iOS)
-/// A concrete `TextSystemInterface` that connects directly to an `NSTextView`/`UITextView`.
+/// A ``TextSystemInterface`` that connects directly to an `NSTextView`/`UITextView`.
```

Comment:
Ok let me backpedal a bit here, maybe:  not sure if this is _also_ a concrete interface in the sense that the one's used under the hood is. Since this is delegating to an even more concrete implementation, maybe call this a "proxy"? Or a "decorator" where the actual decoration is the adaptivity to pick the best-fitting concrete implementation?

Reply:
What is the function of the double-backticks?

Reply:
> Since this is not abstract, I think is really is concrete. But you can always wrap up one (or more!) concrete implementations into another, also concrete one. Don't you think?

I don't have a strong point to make here :)  Changed it back because it's accurate.

------

Backstory:  Skimming the code at first, it sounded like "oh this is a concrete interface, I ought to use this!" especially because it is at the very top of this file (which I attach meaning to). I went as far down as `effectiveInterface` and formed a fake-understanding that this is taking care of things and indeed the way to go. (I didn't recall which type you explicitly mentioned I will need to adjust from 5mins earlier, so I was still in the mode of getting stuff to compile at all.) -- Anyway it surprised me a bit when I found out that `TextStorageSystemInterface` and friends can be used _instead_ just the same. So my intention was to tell readers that this auto-chooser is not a concrete implementation, but a convenience picker/wrapper/adapter/proxy/decorator.